notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -0,0 +1,1051 @@
+{
```

Comment:
<p>change to:</p><p>BUCKET_NAME = "[your-bucket-name]" &nbsp;# @param {type:"string"}</p><p>BUCKET_URI = f"gs://{BUCKET_NAME}"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Created separate variables for bucket name and URI.